### DISCUSSION TITLE
Help a foreigner enjoy this software

### DISCUSSION THREAD

**Discussion by oozan22** [amll-dev/applemusic-like-lyrics#248] [Q&A] [answered]
Hi, will english be supported on the standalone AMLL Player? 

"AMLL Player
A standalone lyrics page player that obtains audio playback information through WebSocket Server."

how to initiate the WS server with a custom port? WHERE is it in the project? couldnt find.

Thanks for the repliers, 
Hope you are having a nice day!

**Comment by Steve-xmh** [comment]
AMLL Player is under refactoring to be a local music player. It may take times so please be patient. And with all of these done I'll try to make a localization feature for other language like English.

**Reply by YaoSiQian** [reply]
> AMLL Player is under refactoring to be a local music player.

Hope to know the last commit of AMLL Player that supports AMLL WS Protocol.

**Reply by Steve-xmh** [reply]
WS Protocol is currently unavailable but is planned to bring it back to current AMLL Player, just be patient.
TBO I forgot what's the last commit of version that support WS Protocol. XD

**Comment by Linho1219** [comment]
<img width="1251" height="868" alt="image" src="https://github.com/user-attachments/assets/a24769e6-5986-47b0-a317-5c9e632b1a42" />

AMLL Player now provides English translation, along with several other languages. And now it also supports WS Protocol.

We're closing this discussion as outdated. If you encounter any other issues, please feel free to open a new one.

**Comment by Linho1219** [comment, marked as answer]
<img width="1251" height="868" alt="image" src="https://github.com/user-attachments/assets/a24769e6-5986-47b0-a317-5c9e632b1a42" />

AMLL Player now provides English translation, along with several other languages. And now it also supports WS Protocol.

We're closing this discussion as outdated. If you encounter any other issues, please feel free to open a new one.
